syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -73,6 +74,20 @@ def __init__(
                     "Unsupported arg value for dtype. Use dtype='long' or dtype='int'."
                 )
 
+    def __mod__(self, divisor: Union[int, torch.Tensor]) -> torch.Tensor:
+        """
+        Define the modulo operation over object instances.
+        """
+        result = self.copy()
+        if isinstance(divisor, int):
+            scaled_divisor = divisor * (self.base ** self.precision_fractional)
+            result.child = result.child % scaled_divisor
+        elif isinstance(divisor, FixedPrecisionTensor):
```

Comment:
calling a function on a tensor is supposed to modify the child without explicitly mentioning it, for instance, "result %= divisor" should do the same, and it's more readable.

Reply:
Agree. I'll change that.

Reply:
Actually, `result %= divisor` ends up being an infinite recursion. So I left it as it was: working with the `.child`. If there's a better way to avoid the recursion without referencing the child please let me know.

Reply:
hmm.. makes sense, let's keep it that way